must-gather/must-gather-requirements.md

```diff
@@ -0,0 +1,51 @@
+# <center>Requirements for OpenShift Sandboxed Containers (OSC) must-gather</center>
+
+
+### Usage
+The kataconfig must have `logLevel: debug` set before running `must-gather`.
+
+OSC `must-gather` should gather all OCS information and logs needed for debugging in a directory
+```sh
+oc adm must-gather --image=registry.redhat.io/openshift-sandboxed-containers/osc-must-gather-rhel9:latest
+```
+Data about other parts of the cluster is gathered with `oc adm must-gather`. Run `oc adm must-gather -h` to see more options.
+
+### Openshift Sandboxed Containers
+Kata runtime is the `containerd-shim-kata-v2` process that talks to the kata agent in the VM.
+See also the [Official 1.5 documentation](https://access.redhat.com/documentation/en-us/openshift_sandboxed_containers/1.5/html-single/openshift_sandboxed_containers_user_guide/index#troubleshooting-sandboxed-containers)
+
+#### Gathered Data
+- Resource definitions
+- Service logs
+- All namespaces and child objects with OSC resources
+- All OSC custom resource definitions (CRDs)
+  - sandboxed-containers/namespaces/openshift-sandboxed-containers-operator/**_*_**\_description
+- versions in nodes/**_nodename_**/**_nodename_**/version
```

Comment:
Per curiosity, what is the motivation to have nodename in bold ?

Reply:
Because it varies.  For my cluster today, the node names will be
tbuskey-240109-2-d9zsw-master-0
tbuskey-240109-2-d9zsw-master-1
tbuskey-240109-2-d9zsw-master-2
tbuskey-240109-2-d9zsw-worker-eastus1-82g44
tbuskey-240109-2-d9zsw-worker-eastus2-s5d6r
tbuskey-240109-2-d9zsw-worker-eastus3-6xdl9